elements/lisk-p2p/src/utils/misc.ts

```diff
@@ -122,14 +120,14 @@ export const getNetgroup = (address: string, secret: number): number => {
 };
 
 // TODO: Source address to be included in hash for later version
-export const getBucket = (options: {
+export const getBucketId = (options: {
 	readonly secret: number;
 	readonly peerType: PEER_TYPE;
 	readonly targetAddress: string;
+	readonly bucketCount: number;
 }): number => {
-	const { secret, targetAddress, peerType } = options;
+	const { secret, targetAddress, peerType, bucketCount } = options;
 	const firstMod = peerType === PEER_TYPE.NEW_PEER ? BYTES_16 : BYTES_4;
-	const secondMod = peerType === PEER_TYPE.NEW_PEER ? BYTES_128 : BYTES_64;
```

Comment:
shouldn't this be used instead of bucketCount?

Reply:
The PeerList expects the `getBucket` function to be able to work with a variable number of buckets, `secondMod` represents the number of buckets; there is no need to couple the number of buckets to the kind of peer list (this could be configured later). The `firstMod` part is not really necessary; the bucket system could work fine without it but it helps to make sure that the two lists (tried vs new) map to different buckets compared to each other (to make it harder to predict one list from the other).